docs/source/reference/command_line.rst

```diff
@@ -71,6 +71,7 @@ Below is a list with all available subcommands.
       --help  Show this message and exit.
 
     Commands:
+      create     Create a new code.
```

Comment:
If this will be the replacement of the code setup, do we need to deprecate it?

Reply:
Yeah, I think we should. Will update it

Reply:
Deprecated